modular_skyrat/modules/customization/modules/clothing/neck/collars.dm

```diff
@@ -4,132 +4,133 @@
 /datum/component/storage/concrete/pockets/small/collar/Initialize()
 	. = ..()
 	can_hold = typecacheof(list(
-	/obj/item/food/cookie,
-	/obj/item/food/cookie/sugar))
+	/obj/item/food/cookie))
 
 /datum/component/storage/concrete/pockets/small/collar/locked/Initialize()
 	. = ..()
 	can_hold = typecacheof(list(
 	/obj/item/food/cookie,
-	/obj/item/food/cookie/sugar,
 	/obj/item/key/collar))
 
 /obj/item/clothing/neck/human_petcollar
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/neck.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/neck.dmi'
 	name = "pet collar"
-	desc = "It's for pets. Though you probably could wear it yourself, you'd doubtless be the subject of ridicule. It seems to be made out of a polychromic material."
-	icon_state = "petcollar_poly"
+	desc = "It's for pets. Though you probably could wear it yourself, you'd doubtless be the subject of ridicule."
+	icon_state = "pet"
+	greyscale_config = /datum/greyscale_config/collar/pet
+	greyscale_config_worn = /datum/greyscale_config/collar/pet/worn
+	greyscale_colors = "#44BBEE#FFCC00"
+	flags_1 = IS_PLAYER_COLORABLE_1
 	alternate_worn_layer = UNDER_SUIT_LAYER
 	pocket_storage_component_path = /datum/component/storage/concrete/pockets/small/collar
-	var/is_polychromic = TRUE
-	var/poly_colors = list("#00BBBB", "#FFCC00", "#FFFFFF")
+	/// What's the name on the tag, if any?
 	var/tagname = null
+	/// What treat item spawns inside the collar?
 	var/treat_path = /obj/item/food/cookie
 
 /obj/item/clothing/neck/human_petcollar/Initialize()
 	. = ..()
 	if(treat_path)
 		new treat_path(src)
 
-/obj/item/clothing/neck/human_petcollar/ComponentInitialize()
-	. = ..()
-	if(is_polychromic)
-		AddElement(/datum/element/polychromic, poly_colors)
-
 /obj/item/clothing/neck/human_petcollar/attack_self(mob/user)
 	tagname = stripped_input(user, "Would you like to change the name on the tag?", "Name your new pet", "Spot", MAX_NAME_LEN)
-	name = "[initial(name)] - [tagname]"
+	if(tagname)
+		name = "[initial(name)] - [tagname]"
```

Comment:
Does this mean you're going to have to erase the "Spot" input every time if you don't want a name, or does that null by default?

Reply:
Collars start with a null name. Before, if you cancelled the input it would give you a blank name i.e. `collar - `. Now, it will keep the null name if there wasn't one, so just `collar`, or the previously input name if there was. 

I considered nulling the name on cancel but I didn't go with that in the end.

Reply:
>It would also be nice to be able to set a name in the loadout rather than having to do it every round, but that's out of scope for this PR (read: I can't be bothered)

couldn't you just copy and paste whatever the plushies use?